docs/src/pages/guides/migrating-to-react-query-3.md

```diff
@@ -65,6 +65,32 @@ const queryClient = new QueryClient({
 })
 ```
 
```

Comment:
We probably need to mention the removal of `queryFnParamsFilter` here as well.

Reply:
Good one, added!